tutormfe/plugin.py

```diff
@@ -107,6 +114,15 @@ def iter_mfes() -> t.Iterable[tuple[str, MFE_ATTRS_TYPE]]:
     yield from get_mfes().items()
 
 
+def iter_plugin_slots(mfe_name: str) -> t.Iterable[tuple[str, str, str]]:
+    """
+    Yield:
+
+        (mfe_name, slot_name, slot_config)
```

Comment:
We are never using `mfe_name` in the templates. Does that mean that we should simply not return it?